2226/2/data_storage.rb

```diff
@@ -0,0 +1,8 @@
+# This module is used for storing data
+module DataStorage
```

Comment:
why module? What if u need more than one storage? 
Create an instance of storage class instead